src/lib.rs

```diff
@@ -119,6 +119,27 @@ pub unsafe extern "C" fn dav1d_version() -> *const c_char {
     rav1d_version().as_ptr().cast()
 }
 
+/**
+ * Get library API version.
+ *
+ * @return A value in the format 0x00XXYYZZ, where XX is the major version,
+ *         YY the minor version, and ZZ the patch version.
+ */
+#[no_mangle]
+#[cold]
+pub unsafe extern "C" fn dav1d_version_api() -> c_uint {
```

Comment:
This can be safe right?  Also, do we need to parse it from a string?

Reply:
The others are all `unsafe` either because we haven't made them safe yet, or because they take raw pointers and need to remain `unsafe`.  This one is just `() -> u32`, so it can be safe.

I think we can for now just define them as `const` `u8`s.  In the future, if we match `rav1d`'s `cargo` version to this, we can use `env!("CARGO_PKG_VERSION_MAJOR")`, `env!("CARGO_PKG_VERSION_MINOR")`, and `env!("CARGO_PKG_VERSION_PATCH")` that `cargo` will set for us.

Reply:
I made the function safe.

I had looked at the `cargo` package version, but wasn't convinced it was the right thing to do.